src/wrappers/vue-wrapper.js

```diff
@@ -29,4 +29,8 @@ export default class VueWrapper extends Wrapper implements BaseWrapper {
 
     logEvents(vm, this._emitted, this._emittedByOrder)
   }
+
+  exists (): boolean {
+    return !!this.vm && !this.vm._isDestroyed
```

Comment:
Can you move this method to wrapper.js and return true if this.isVueComponent is false. A node Wrapper will always exist. If find couldn't find anything, it would return an ErrorWrapper instead